src/routes/products/[productId]/+server.ts

```diff
@@ -4,5 +4,10 @@ import { stripe as stripeClient } from '$lib/utils/stripe';
 export const GET = async ({ params }) => {
 	const { productId } = params;
 	const product = await stripeClient.products.retrieve(productId);
+
+	const price = await stripeClient.prices.retrieve(product.default_price);
+
+	const isSubscription = price.type === 'recurring';
```

Comment:
Can you cast 'recurring' as `stripe.Price.Type` for typesafety and Stripe update wizardry 🪄

Reply:
Updated :)